ts/src/main/scala/ch/epfl/ts/example/SimpleExampleHistFX.scala

```diff
@@ -0,0 +1,78 @@
+package ch.epfl.ts.example
+
+import ch.epfl.ts.component.ComponentBuilder
+import ch.epfl.ts.engine.ForexMarketRules
+import ch.epfl.ts.component.fetch.TrueFxFetcher
+import ch.epfl.ts.component.persist.DummyPersistor
+import ch.epfl.ts.component.fetch.MarketNames
+import ch.epfl.ts.engine.MarketFXSimulator
+import akka.actor.Props
+import ch.epfl.ts.traders.SimpleFXTrader
+import ch.epfl.ts.component.utils.BackLoop
+import ch.epfl.ts.indicators.SmaIndicator
+import scala.reflect.ClassTag
+import ch.epfl.ts.component.fetch.PullFetchComponent
+import ch.epfl.ts.engine.RevenueComputeFX
+import ch.epfl.ts.data.{ Quote, OHLC }
+import ch.epfl.ts.data.Transaction
+import ch.epfl.ts.data.MarketAskOrder
+import ch.epfl.ts.data.MarketBidOrder
+import ch.epfl.ts.indicators.{ OhlcIndicator, MaIndicator, MovingAverage, SMA }
+import ch.epfl.ts.data.Currency
+import ch.epfl.ts.engine.RevenueCompute
+import ch.epfl.ts.component.fetch.HistDataCSVFetcher
+import ch.epfl.ts.traders.SimpleFXTraderWithShorting
+
+object SimpleExampleHistFX {
```

Comment:
I think this is mostly duplicated from another one of the example configurations. Would if be possible to merge them and maybe use boolean flags if there are choices to be made?